src/frontend/src/handler/util.rs

```diff
@@ -87,9 +87,9 @@ fn pg_value_format(data_type: &DataType, d: ScalarRefImpl<'_>, format: bool) ->
     // format == true means BINARY format
     if !format {
         match (data_type, d) {
-            (DataType::Boolean, ScalarRefImpl::Bool(b)) => if b { "t" } else { "f" }.into(),
+            // TODO: introduce ScalarRefImpl::TimestampZ
```

Comment:
Have not decided whether to introduce `ScalarRefImpl::Timestampz` yet. It actually prevents some mis-assumption that `Scalar` contains enough type information, where `DataType` actually needs to be passed as argument together.